x/websocket/endpoint.go

```diff
@@ -111,7 +112,11 @@ func newGorillaConn(wsConn *websocket.Conn) *gorillaConn {
 }
 
 type gorillaConn struct {
-	wsConn        *websocket.Conn
+	wsConn *websocket.Conn
+
+	// websocket.Conn is not safe for concurrent use
```

Comment:
Perhaps add a comment mentioning the bug for future context.

Reply:
Added